src/hotspot/share/jfr/recorder/checkpoint/types/traceid/jfrTraceIdLoadBarrier.hpp

```diff
@@ -68,6 +68,7 @@ class PackageEntry;
  */
 class JfrTraceIdLoadBarrier : AllStatic {
   friend class JfrCheckpointManager;
+  friend class SharedRuntime;
```

Comment:
Don't think we need to involve SharedRuntime.

Reply:
removed.